examples/for-tests/src/App.js

```diff
@@ -697,8 +697,8 @@ function getThirdPartyEmailPasswordConfigs({ disableDefaultImplementation }) {
                 ThirdPartyEmailPassword.Facebook.init(),
                 ThirdPartyEmailPassword.Apple.init(),
                 {
-                    id: "custom",
-                    name: "Custom",
+                    id: "auth0",
+                    name: "Auth0",
                 },
```

Comment:
Do not remove custom.. add another one for auth0. Cause then other tests will fail.